src/components/BasePriority.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <p
+    :class="`${bgColor} w-fit place-self-end rounded-2xl px-4 text-sm text-white sm:px-5 sm:text-base`"
```

Comment:
You can use here `class` and `:class` attributes.
`class` attribute will contain the constant classnames and the `:class` attribute will contain the dynamic variable.